src/transformers/models/whisper/convert_openai_to_hf.py

```diff
@@ -154,6 +201,9 @@ def convert_openai_whisper_to_tfms(checkpoint_path, pytorch_dump_folder_path):
     tie_embeds = True
     ffn_dim = state_dict["decoder.layers.0.fc1.weight"].shape[0]
 
+    # a hacky way to properly set up the bos/eos/pad token ids in the model
+    endoftext_id = 50257 if dimensions["n_vocab"] > 51865 else 50256
+
     config = WhisperConfig(
```

Comment:
Nice! The only missing config to update here is the `decoder_start_token_id` (`endoftext_id + 1`)

Reply:
@sanchit-gandhi done